src/App.tsx

```diff
@@ -15,71 +15,84 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string;
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state: State = {
+    selectedGood: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  handleClick(item: string) {
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+    item === this.state.selectedGood
+      ? this.setState({
+        selectedGood: '',
+      })
+      : this.setState({
+        selectedGood: item,
+      });
+  }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+  render() {
+    return (
+      <main className="section container">
+        {this.state.selectedGood ? (
```

Comment:
Better destructure selectedGood from state to make your code a bit cleaner
`const { selectedGood } = this.state;`